packages/terminal-extension/src/index.ts

```diff
@@ -113,7 +113,9 @@ function activate(app: JupyterLab, services: IServiceManager, mainMenu: IMainMen
     CommandIDs.toggleTheme
   ].forEach(command => {
     palette.addItem({ command, category });
-    menu.addItem({ command });
+    if (command !== CommandIDs.createNew) {
+      menu.addItem({ command });
+    }
```

Comment:
I think we will need to think through how to accomplish this in a clean and consistent manner. We may end up adding "New *" items to all the menus, but let's see where we can get...